packages/core/src/__tests__/memdb.test.ts

```diff
@@ -19,35 +19,33 @@ import { ModelDb } from '../memdb'
 import core from '../component'
 import { Domain } from '../classes'
 
-describe('hierarchy', () => {
+const txes = require('./core.tx.json') // eslint-disable-line
 
+describe('hierarchy', () => {
   it('should build hierarchy', async () => {
-    const txes = ((await import('./core.tx.json')) as any).default as Tx[]
     const hierarchy = new Hierarchy()
     for (const tx of txes) hierarchy.tx(tx)
-    const ancestors = hierarchy.getAncestors(core.class.TxCreateObject)
+    const ancestors = hierarchy.getAncestors(core.class.TxCreateDoc)
     expect(ancestors).toContain(core.class.Tx)
   })
 
   it('should query model', async () => {
-    const txes = ((await import('./core.tx.json')) as any).default as Tx[]
     const hierarchy = new Hierarchy()
     for (const tx of txes) hierarchy.tx(tx)
     const model = new ModelDb(hierarchy)
-    for (const tx of txes) model.tx(tx)
+    for (const tx of txes) model.tx(tx) // eslint-disable-line
     const result = await model.findAll(core.class.Class, {})
     expect(result.length).toBe(3)
   })
 
   it('should query model with params', async () => {
-    const txes = ((await import('./core.tx.json')) as any).default as Tx[]
     const hierarchy = new Hierarchy()
     for (const tx of txes) hierarchy.tx(tx)
     const model = new ModelDb(hierarchy)
-    for (const tx of txes) model.tx(tx)
-    const first = await model.findAll(core.class.Class, { _id: txes[0].objectId, domain: "model" as Domain })
+    for (const tx of txes) model.tx(tx) // eslint-disable-line
```

Comment:
Fix txes and remove // eslint-disable-line

Reply:
It is not related to txes rule is following:

https://github.com/typescript-eslint/typescript-eslint/blob/v4.23.0/packages/eslint-plugin/docs/rules/no-floating-promises.md